apps/gp2-server/serverless.ts

```diff
@@ -23,8 +23,8 @@ assert.ok(
 
 const stage = process.env.SLS_STAGE!;
 const region = process.env.AWS_REGION as AWS['provider']['region'];
-const acmCertificateArn = process.env.AWS_ACM_CERTIFICATE_ARN!;
-const hostname = process.env.ASAP_HOSTNAME!;
+const gp2AwsAcmCertificateArn = process.env.GP2_AWS_ACM_CERTIFICATE_ARN!;
```

Comment:
Should be variable name remain acmCertificateArn but if reads it's value from GP2_AWS_ACM_CERTIFICATE_ARN

Reply:
im not sure actually, should it?